tests/dummy/app/routes/nodes/detail/draft-registrations.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+import permissions from 'ember-osf/const/permissions';
+
+export default Ember.Route.extend({
+    model() {
+        let node = this.modelFor('nodes.detail');
+        let drafts = node.get('draftRegistrations');
+        return Ember.RSVP.hash({
```

Comment:
I agree with Andy here. We've experienced a lot of unexpected behavior when using hashes in the model hook. In this case, using `setupController` to set the `node` property on the controller is the preferred solution here.